[2024-10-15T05:11:00.209+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-15T05:11:00.255+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: eia_data_pipeline.call_stored_procedure scheduled__2024-10-14T00:00:00+00:00 [queued]>
[2024-10-15T05:11:00.283+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: eia_data_pipeline.call_stored_procedure scheduled__2024-10-14T00:00:00+00:00 [queued]>
[2024-10-15T05:11:00.285+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-15T05:11:00.324+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): call_stored_procedure> on 2024-10-14 00:00:00+00:00
[2024-10-15T05:11:00.368+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=116) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T05:11:00.391+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'eia_data_pipeline', 'call_stored_procedure', 'scheduled__2024-10-14T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/new_dag.py', '--cfg-path', '/tmp/tmpt7ch9xl8']
[2024-10-15T05:11:00.403+0000] {standard_task_runner.py:72} INFO - Started process 126 to run task
[2024-10-15T05:11:00.404+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask call_stored_procedure
[2024-10-15T05:11:01.061+0000] {task_command.py:467} INFO - Running <TaskInstance: eia_data_pipeline.call_stored_procedure scheduled__2024-10-14T00:00:00+00:00 [running]> on host 7aa864b16460
[2024-10-15T05:11:01.686+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='eia_data_pipeline' AIRFLOW_CTX_TASK_ID='call_stored_procedure' AIRFLOW_CTX_EXECUTION_DATE='2024-10-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-14T00:00:00+00:00'
[2024-10-15T05:11:01.701+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-15T05:11:01.712+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-15T05:11:01.719+0000] {logging_mixin.py:190} INFO - Current task name:call_stored_procedure state:running start_date:2024-10-15 05:11:00.257622+00:00
[2024-10-15T05:11:01.741+0000] {logging_mixin.py:190} INFO - Dag name:eia_data_pipeline and current dag run status:running
[2024-10-15T05:11:01.744+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-15T05:11:12.141+0000] {logging_mixin.py:190} INFO - Stored procedure executed successfully.
[2024-10-15T05:11:12.192+0000] {logging_mixin.py:190} INFO - MySQL connection is closed
[2024-10-15T05:11:12.213+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-15T05:11:12.352+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-15T05:11:12.366+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=eia_data_pipeline, task_id=call_stored_procedure, run_id=scheduled__2024-10-14T00:00:00+00:00, execution_date=20241014T000000, start_date=20241015T051100, end_date=20241015T051112
[2024-10-15T05:11:12.439+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-15T05:11:12.443+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-15T05:11:12.448+0000] {logging_mixin.py:190} INFO - Dag name:eia_data_pipeline queued_at:2024-10-15 05:08:38.006493+00:00
[2024-10-15T05:11:12.457+0000] {logging_mixin.py:190} INFO - Task hostname:7aa864b16460 operator:PythonOperator
[2024-10-15T05:11:12.602+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-15T05:11:12.759+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T05:11:12.772+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
